code/game/machinery/computer/crew.dm

```diff
@@ -120,14 +120,17 @@ GLOBAL_DATUM_INIT(crewmonitor, /datum/crewmonitor, new)
 	var/life_status
 
 	for(var/mob/living/carbon/human/H in GLOB.carbon_list)
+		var/nanite_sensors = FALSE
+		if(H in SSnanites.nanite_monitored_mobs)

```

Comment:
please refactor this into two loops, one to build a list of targets as SSnanites.nanite_monitored_mobs | carbon_list mobs that meet requirements and the other to loop through the list

Reply:
Would it be better to do it with a signal made specifically for this, or would it be too snowflake?